mlptrain/training/active.py

```diff
@@ -69,7 +70,8 @@ def train(mlp:               'mlptrain.potentials._base.MLPotential',
 
         init_configs: (gt.ConfigurationSet) A set of configurations from
                       which to start the active learning from
-
+        
+        torch: (bool) if want to share tensor between processes, set torch == True
```

Comment:
Slightly unclear what this means(?). Do you mean if you want to share tensor calculations between processes, use torch == True? Also need capitalisation at the start of the sentence and don't need to specify how to set it in the doc string.

Reply:
I do not understand what you mean by tensor calculation.

Reply:
The doc string isn't clear to me what the bool represents or does, so needs to be clarified. What do you mean by share tensor between processes?